### PR TITLE
docs(buttons): improve docs for icons and action links

### PR DESCRIPTION
**Pull request by glynnis**
Closes https://github.com/rackerlabs/encore-ui/issues/787

### CODE REVIEW

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/796/gallery.html)

File: demo/styleguide/buttons.html [hunk 1]
```diff
@@ -38,8 +38,11 @@ <h2 id="colors">Colors</h2>
...
+</p>
+
+<p>For the majority of Encore, actionable links are used in two places&#8212;in the upper right hand section of a details page, flushed right with the <code>page-actions</code> and <code>actions-area</code> classes, or in the rxActionMenu directive.</p>
+<p>Below we have included icons that have used throughout Encore, as well as the actions associated with the icons. Please use as a reference for which icons are already in use and should not be duplicated.</p>
```
Comment (parlarjb) [inline demo/styleguide/buttons.html]: "Below we have included icons that have BEEN used through..."

Comment (glynnis) [inline demo/styleguide/buttons.html]: Done!

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/796/gallery.html)

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**